spec/support/fake_bridge_troll.rb

```diff
@@ -0,0 +1,9 @@
+require "sinatra/base"
+
+class FakeBridgeTroll < Sinatra::Base
+  get "/users/1/events" do
+    status 200
+    content_type :json
+    '{"event_count": 15}'
+  end
+end
```

Comment:
This seems a roundabout way to mock. Why not use vcr, or plain fakeweb or webmock?